packages/components/src/components/AutoScalingInput/AutoScalingInput.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import { AutoScalingInput } from './AutoScalingInput';
+
+const Wrapper = styled.div`
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+`;
+
+const BorderAutoScalingInput = styled(AutoScalingInput)`
+    padding: 1px 5px 1px 5px;
+    border-radius: 3px;
+    border-style: solid;
+    border-width: 1px;
+`;
+
+const BorderlessAutoScalingInput = styled(AutoScalingInput)`
+    padding: 1px 5px 1px 5px;
+    border-style: solid;
+    border-width: 0px;
+    background-color: #ccc;
+`;
+
+export default {
+    title: 'Misc/AutoScalingInput',
+};
+
+export const Box = {
```

Comment:
<img width="733" alt="Screenshot 2024-01-08 at 13 42 33" src="https://github.com/trezor/trezor-suite/assets/858321/1af54ff6-15fb-48d2-a6dd-deddc6af2d08">

Reply:
I think AutoScalingInput doesn't have to be expandable.

Reply:
I don't know what you mean here. Please hit me on Slack.

Reply:
just change 
```
export const Box = {
```

to 

```
export const AutoScalingInput = { // or with different in case of different name
```

if it has the same name as `title` segment in default export it will not create tree structure